src/cf-forms/src/molecules/form-fields.less

```diff
@@ -123,7 +131,7 @@
         .a-radio {
             &:focus + .a-label,
             &:checked + .a-label {
-                outline: 1px solid @input-border__focused;
+                box-shadow: 0px 0px 0px 1px @input-border__focused inset;
```

Comment:
Ah, I had mixed messages on this. Do we definitely have design docs that the selected size should be larger? Meaning a line of vertically stacked radio buttons will not have an aligned left/right edge when one is selected.